collections/hrsi_fsc.yaml

```diff
@@ -0,0 +1,80 @@
+Name: 
+High-Resolution Snow & Ice Monitoring; Fractional Snow Cover
+Description: 
+The High-Resolution Snow & Ice Monitoring service (HR-S&I) is part of the Copernicus Land Monitoring Service (CLMS). The snow aspect of the service provides products 
+measuring Snow cover (FSC, FSTOC, FSCOG, GFSC), Snow state conditions (WDS, SWS) and persistent snow area (PSA). There are also ice products that measure ice cover 
+(RLIE) and aggregated river and lake ice extent (ARLIE).
+
+The Fractional Snow Cover (FSC) product provides the snow fraction at the Top Of Canopy (FSCTOC) and On Ground (FSCOG). NDSI is also provided as part of this service.
+AdditionalInfoExternal:
+    Title: Additional info 
+    Path: /README.MD 
+Image: /thumbnail.png       
```

Comment:
Create a nice square thumbnail image from EO browser not more than (400*400 ) 

save in the respective folder  and provide the relative  path `Image: hrsi_fsc/thumbnail.png `

same thing to the rest of the collections